### PR TITLE
Align custom value with TC plugin

### PR DESCRIPTION
**Pull request by sorin-florea**
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

### CODE REVIEW

File: src/main/resources/hudson/plugins/gradle/injection/init-script.gradle [hunk 1]
```diff
@@ -48,7 +48,7 @@ def BUILD_SCAN_PLUGIN_CLASS = 'com.gradle.scan.plugin.BuildScanPlugin'
 def GRADLE_ENTERPRISE_PLUGIN_ID = 'com.gradle.enterprise'
 def GRADLE_ENTERPRISE_PLUGIN_CLASS = 'com.gradle.enterprise.gradleplugin.GradleEnterprisePlugin'
 def GRADLE_ENTERPRISE_EXTENSION_CLASS = 'com.gradle.enterprise.gradleplugin.GradleEnterpriseExtension'
-def CI_AUTO_INJECTION_CUSTOM_VALUE_NAME = 'CI-auto-injection'
+def CI_AUTO_INJECTION_CUSTOM_VALUE_NAME = 'CI auto injection'
```
Comment (guylabs) [inline src/main/resources/hudson/plugins/gradle/injection/init-script.gradle:51]:
```suggestion
def CI_AUTO_INJECTION_CUSTOM_VALUE_NAME = 'CI auto-injection'
```

**Review by guylabs — CHANGES_REQUESTED** [review]

**Review by facewindu — APPROVED** [review]
nothing more than what Guy said

File: src/main/resources/hudson/plugins/gradle/injection/init-script.gradle (hunk 1 above)
Comment (sorin-florea) [reply src/main/resources/hudson/plugins/gradle/injection/init-script.gradle:51]: See https://gradle.slack.com/archives/C03G38H88P4/p1666364657763789?thread_ts=1666352048.466839&cid=C03G38H88P4

Comment (guylabs) [reply src/main/resources/hudson/plugins/gradle/injection/init-script.gradle:51]: Revoking this as discussed.

**Review by guylabs — APPROVED** [review]

**Review by c00ler — APPROVED** [review]

**Review by welandaz — APPROVED** [review]

**Pull request merged**